qa/Vagrantfile

```diff
@@ -0,0 +1,27 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+require_relative "./platform_config.rb"
+
+Vagrant.configure(2) do |config|
+  platforms = PlatformConfig.new
```

Comment:
I suggest that if we ever have the need a different settings, we make the changes. But right now since everything is the same we should just use the dry up version instead.